src/Place/ReadModel/JSONLD/PlaceLDProjector.php

```diff
@@ -292,7 +308,23 @@ function ($term) {
         $placeLd->terms = $terms;
 
         $this->repository->save($document->withBody($placeLd));
+    }
 
+    /**
+     * @param GeoCoordinatesUpdated $geoCoordinatesUpdated
+     */
+    protected function applyGeoCoordinatesUpdated(GeoCoordinatesUpdated $geoCoordinatesUpdated)
+    {
+        $document = $this->loadPlaceDocumentFromRepository($geoCoordinatesUpdated);
+
+        $placeLd = $document->getBody();
+
+        $placeLd->geo = (object) [
+            'latitude' => $geoCoordinatesUpdated->getCoordinates()->getLatitude()->toDouble(),
+            'longitude' => $geoCoordinatesUpdated->getCoordinates()->getLongitude()->toDouble(),
```

Comment:
Dit zou eigenlijk "lat" & "lon" moeten zijn om compatibel te zijn met geo points in Elasticsearch: https://www.elastic.co/guide/en/elasticsearch/guide/current/lat-lon-formats.html

Volgens schema.org zou het wel "latitude" en "longitude" moeten zijn, maar dat lijkt me minder belangrijk als de search aan de praat krijgen.

Alternatief is dat we bij indexering op de search service deze omzetten met een soort van transformer.